src/test/parsers/minecraft/coord/coord_base.test.ts

```diff
@@ -0,0 +1,171 @@
+import * as assert from "assert";
+import { StringReader } from "../../../../brigadier_components/string_reader";
+import { isSuccessful } from "../../../../misc_functions";
+import {
+    CoordBaseParser,
+    CoordRules
+} from "../../../../parsers/minecraft/coord/coord_base";
+import { ReturnedInfo } from "../../../../types";
+import { assertErrors } from "../../../assertions";
+
+const toString = (v: CoordRules) =>
+    `{count: ${v.count}, float: ${v.float}, local: ${v.local}}`;
+
+describe("Coordinate tests", () => {
+    describe("parse()", () => {
+        ([
```

Comment:
I'm sure in function these tests are fine, but please could you use the `testParser` api and seperate explicit `it`s for each test.